src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
                 },
                 {
                     "term": "Impfzertifikat",
                     "anchor": "vaccination_certificate",
-                    "description": "Das Impfzertifikat ist ein <a href='#glossary_digital_certificate'>digitales Zertifikat</a>, das einer Person ausgestellt wird, die eine Impfung mit einem in der EU zugelassenen Impfstoff wahrgenommen haben. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis). <br/><br/> Das Impfzertifikat stellt eine digitale Form für einen <a href='#glossary_vaccination_proof'>Impfnachweis</a> dar."
+                    "description": "Bestätigung über eine durchgeführte Impfung. Das digitale Impfzertifikat wird durch das Scannen von QR-Codes auf den vom jeweiligen medizinischen Personal erhaltenen Dokumenten erstellt. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis).<br></br>Das Impfzertifikat stellt eine digitale Form für einen <a href='#glossary_proof_of_vaccination'>Impfnachweis</a> dar."
+                },
+                {
+                    "term": "Impfstatus",
+                    "anchor": "vaccination_status",
+                    "description": "Zum Impfstatus gehören Informationen wie die Anzahl an erfolgten Impfungen, der verwendete Impfstoff und die Gültigkeit der Impfung."
+                },
+                {
+                    "term": "iOS",
+                    "anchor": "ios",
+                    "description": "iOS ist ein von Apple entwickeltes mobiles Betriebssystem für das iPhone, iPod touch und ehemalig auch iPad. Neben <a href='#glossary_android'>Android</a> ist es eines der beiden Betriebssysteme, die die Corona-Warn-App unterstützt."
+                }
+            ],
+            "K": [
+                {
+                    "term": "Kontakttagebuch",
+                    "anchor": "contact_diary",
+                    "description": "Im Kontakttagebuch lassen sich Treffen/Kontakt einfach pro Tag der vergangenen 14 Tage festhalten. Darüber hinaus werden hier auch u.a. Events oder Risikowarnungen angezeigt. Die Informationen können über eine Export-Funktion des Kontakttagebuchs manuell mit dem Gesundheitsamt geteilt werden."

```

Comment:
Treffen und Kontakte

Reply:
Will be changed in the next commit, thanks!